barretenberg/cpp/src/barretenberg/honk/proof_system/protogalaxy_prover.hpp

```diff
@@ -51,6 +51,21 @@ template <class ProverInstances_> class ProtoGalaxyProver_ {
      */
     void prepare_for_folding();
 
```

Comment:
Will you add a test showing that this beta function produces a power of beta in the special case that the beta vector is a vector of powers?